### ISSUE TITLE
Shortcuts interfere with website-specific shortcuts

### ISSUE THREAD

**Issue by tranquil-tr0** [zen-browser/desktop#3031]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

In Google Docs, copy and paste formatting can be done through the hotkeys ctrl-alt-c ctrl-alt-v, respectively. This interferes with Zen Browser's tab tiling shortcuts, though, resulting in both the website and browser to respond to hotkeys. This also does not happen with ctrl-alt-s, which results in the browser failing to recognize the shortcut, which I would expect to be expected behavior.

[EDIT] To clarify, the issue for me was that:
a) with ctrl-alt-c ctrl-alt-v, the **browser** responded to the hotkeys **while** passing them to the site.
b) this was inconsistent with ctrl-alt-s, where this was not the case, and hotkeys acted as expected (sent **only** to site, not browser)
The issue was not that the browser sent hotkeys to the site instead, and I actually prefer this behavior personally. The issue was that the browser sometimes did **both** [END OF EDIT]

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.19

### What platform are you seeing the problem on?

Windows

**Comment by JSGRD22** [comment]
For me, if there is a conflict, websites will usually override the browser's shortcuts instead, which is also a problem.

**Comment by nekeo** [comment]
On some browsers, you can choose which shortcut should take priority: the browser or the site
This could be a good solution

**Comment by Richard-Woessner** [comment]
Hey Im on mac can you screenshot which hotkey is default set to shift-ctrl-c and shift-ctrl-v. Im trying to recreate this issue

**Comment by Smooth-E** [comment]
> On some browsers, you can choose which shortcut should take priority: the browser or the site This could be a good solution

This issue is probably related to [the following enhancement request (discussion)](https://github.com/zen-browser/desktop/discussions/3039).

It also contains a possible workaround:

> type `about:config` into URL bar, then search for `permissions.default.shortcuts` and change the value to 2 (default value is 0).
> Thank you to annonymous discord member for helping find this.

**Comment by tranquil-tr0** [comment]
> For me, if there is a conflict, websites will usually override the browser's shortcuts instead, which is also a problem.

Yup! That definitely comes up more for me as well, but I generally make us of website shortcuts more often (for a given site) and because Zen has the great gift of actually letting me choose shortcuts. It was also because this was inconsistent, since I had assumed that it would always result in sites taking priority because of ctrl-alt-s, while ctrl-alt-v gave the opposite result.

**Comment by tranquil-tr0** [comment]
> Hey Im on mac can you screenshot which hotkey is default set to shift-ctrl-c and shift-ctrl-v. Im trying to recreate this issue

![image](https://github.com/user-attachments/assets/4812ad1a-85eb-4701-9ac6-d3b872d64c56)
![image](https://github.com/user-attachments/assets/07cf44c4-4dc6-421d-a2f1-78c3a95a728a)
![image](https://github.com/user-attachments/assets/0efd3000-8841-4d00-bc03-6aa12bdc6626)
EDIT: google docs screenshot on top, Zen screenshots on the bottom

**Comment by codexous** [comment]
can find a solution/workaround here: https://github.com/zen-browser/desktop/discussions/3039

I don't believe it's a bug, just an oversight. Zen will _always_ prefer site shortcuts over the ones you assign in settings. works on discord, chatgpt, etc.

**Comment by tranquil-tr0** [comment]
> I don't believe it's a bug, just an oversight. Zen will _always_ prefer site shortcuts over the ones you assign in settings. works on discord, chatgpt, etc.

My issue is that Zen didn't do that with ctrl-alt-c & ctrl-alt-v
EDIT: I think I wasn't clear enough initially, so I updated my issue text

**Comment by codexous** [comment]
alr, understood o7

**Comment by tranquil-tr0** [comment]
This issue is not fixed, but it does not happen on Linux. Original report is updated.

**Comment by mr-cheffy** [comment]
This is expected, it's part of the web standard

**Comment by tranquil-tr0** [comment]
I'm not super familiar with web standards, but I assume they would be platform-consistent? Described behavior (shortcut registered by browser and webpage) only occurs on my Windows machine, but not Linux, where behavior is only registered by webpage. I would assume whatever the standard is, that it would be the same on both machines.

**Comment by watermelvin** [comment]
Hi friends, upon reading https://bugzilla.mozilla.org/show_bug.cgi?id=1445942, I discovered that setting permissions.default.shortcuts in about:config to 3 instead of its default of 0 or the suggested 2 allows you override site specific shortcuts, while also not breaking the backspace key (at least for me on google docs.)

**Comment by tranquil-tr0** [comment]
While interesting, this is unfortunately not the problem I was facing. I hope this helps someone out in the future though!
Based on my reading of the Firefox code, the `permissions.default.shortcuts` only has values 0, 1, 2, 3. After testing all four values, the behavior of having keyboard shortcuts registered by both the browser and webpage has not been solved, nor does it change the what I believe to be correct behavior on Linux.
